back-end/validations/validName.js

```diff
@@ -0,0 +1,18 @@
+/** @format */
+
+const validName = (snack) => {
+	let splitNameArray = snack.name.split(' ');
+	console.log(splitNameArray);
+	let casedName = splitNameArray
+		.map((word) => {
+			if (word.length > 2) {
```

Comment:
could you solve this with a Regular Expression?

Reply:
@JedeBrahma Let's discuss this! I feel like I need to really learn regular expressions.

Reply:
This is Jede's baby. She wrote the code that solved the problem. Would love to learn how to do it with regex as well.